pkg/persistence/account/internal/types/types.go

```diff
@@ -59,17 +59,17 @@ type (
 		// Account level permissions and policies that apply to users in this group
 		Account *Account `yaml:"account,omitempty" json:"account,omitempty" jsonschema:"description=Account level permissions and policies that apply to users in this group."`
 		// Environment level permissions and policies that apply to users in this group
-		Environment []Environment `yaml:"environment,omitempty" json:"environment,omitempty" jsonschema:"description=Environment level permissions and policies that apply to users in this group."`
+		Environment []Environment `yaml:"environments,omitempty" json:"environments,omitempty" jsonschema:"description=Environment level permissions and policies that apply to users in this group."`
```

Comment:
nit: Would make sense to make the struct fields for Env and MZ plural too IMO, they are slices anyway.